ui/packages/consul-peerings/app/templates/dc/peers/index.hbs

```diff
@@ -126,12 +126,12 @@ as |sort filters items|}}
                 <BlockSlot @name="actions">
                   <li class="docs-link">
                     {{!-- what's the docs for peering?--}}
-                    <a href="https://www.consul.io/docs/agent/kv" rel="noopener noreferrer" target="_blank">
+                    <a href="https://www.consul.io/docs/connect/cluster-peering" rel="noopener noreferrer" target="_blank">
```

Comment:
Oh just noticed, these need to use either `env 'CONSUL_DOCS_URL'` and `env CONSUL_DOCS_LEARN_URL`, or the `t` helper.

Guessing we don't have an i18n in this file yet? If so probably just go with the env stuff if thats easier. Will find you 2 examples already in the codebase in a sec.

Also, do you know where you copy pasted this from, consul.io shouldn't be mentioned anywhere in the codebase apart from the set the single env var?